### PR TITLE
UDP-451 bumping to new UDP version 11.0.1

### PR DESCRIPTION
**Pull request by alebastrov**
(major feature there is database changes)

### CODE REVIEW

File: boilerplate/Jenkinsfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ pipeline {
     parameters {
```
Comment (treaz) [inline boilerplate/Jenkinsfile]: we don't use Jenkinsfiles anymore. Let's just get rid of it

File: webapp-java/dxa-module-dynamic-documentation/pom.xml [hunk 2]
```diff
@@ -97,7 +97,7 @@
                                 <artifactItem>
                                     <groupId>${project.groupId}</groupId>
                                     <artifactId>dynamic-documentation-boilerplate</artifactId>
-                                    <version>${project.version}</version>
+                                    <version>2.2.11</version>
```
Comment (treaz) [inline webapp-java/dxa-module-dynamic-documentation/pom.xml:100]: why not 2.2.12-SNAPSHOT?

File: webapp-java/pom.xml [hunk 3]
```diff
@@ -6,8 +6,8 @@
     <parent>
         <groupId>com.sdl.dxa</groupId>
         <artifactId>dxa-oss-parent</artifactId>
-        <version>2.2.11</version>
-        <relativePath></relativePath>
+        <version>2.2.12-SNAPSHOT</version>
+<!--        <relativePath></relativePath>-->
```
Comment (treaz) [inline webapp-java/pom.xml]: remove this commented out piece, please

File: boilerplate/Jenkinsfile (hunk 1 above)
Comment (alebastrov) [reply boilerplate/Jenkinsfile]: done in develop

File: webapp-java/dxa-module-dynamic-documentation/pom.xml (hunk 2 above)
Comment (alebastrov) [reply webapp-java/dxa-module-dynamic-documentation/pom.xml:100]: fixed

File: webapp-java/pom.xml (hunk 3 above)
Comment (alebastrov) [reply webapp-java/pom.xml]: done

**Pull request merged**